### PR TITLE
Firepower Integration - add Get&Update url list for group object and get deployment status

### PR DESCRIPTION
**Pull request by morita39**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [x

 ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by morita39** [comment]
Hi
I am not very familiar with Git pull request, compare file output shows looks there are a lot of changes ,but no.
I tested the code in XSOAR 6.0.1 and works fine.

**Comment by reutshal** [comment]
@morita39 Let's schedule a short meeting to better understand the use case here and to order this PR.
Can you please share your availability? thanks

**Comment by morita39** [comment]
> @morita39 Let's schedule a short meeting to better understand the use case here and to order this PR.
> Can you please share your availability? thanks

**Comment by dantavori** [comment]
@morita39 I fixed the indentation issue. I also added some changes required in your pull request:
1. Every integration command and its details is defined in the integration **yml** file - added the new argument of the "get deployment status" and the other two new commands.
2. A **release notes** file, which contains the update details (1.0.3).
3. Updated the pack version in the metadata file to 1.0.3.

Before I review everything, please:
1. Make sure the fixes I made in the py file are correct and I didn't miss anything.
2. Complete any missing description in the yml file - search for all the "TODO"s. Also verify the rest of the details in the yml file are correct (e.g. whether or not an argument is required, isArray, etc).
3. Make sure the release notes file I added is correct.
4. Test this version of the integration locally on your env (contact me on Slack if you need help in this), to make sure what we merge is working.
5. Generate the readme file with the updates (after you updated all the descriptions in the yml file).

**Comment by morita39** [comment]
> @morita39 I fixed the indentation issue. I also added some changes required in your pull request:
> 
> 1. Every integration command and its details is defined in the integration **yml** file - added the new argument of the "get deployment status" and the other two new commands.
> 2. A **release notes** file, which contains the update details (1.0.3).
> 3. Updated the pack version in the metadata file to 1.0.3.
> 
> Before I review everything, please:
> 
> 1. Make sure the fixes I made in the py file are correct and I didn't miss anything.
> 2. Complete any missing description in the yml file - search for all the "TODO"s. Also verify the rest of the details in the yml file are correct (e.g. whether or not an argument is required, isArray, etc).
> 3. Make sure the release notes file I added is correct.
> 4. Test this version of the integration locally on your env (contact me on Slack if you need help in this), to make sure what we merge is working.
> 5. Generate the readme file with the updates (after you updated all the descriptions in the yml file).

Thank you for reaching out to me. I confirmed 
1. py file looks Okay. 
2. I Updated yml file "TODO" areas. please kindly check
3. The release note is corrected wording. thanks 
4. I will happy to do

File: Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.yml [hunk 1]
```diff
@@ -2255,10 +2279,70 @@ script:
...
     outputs:
     - contextPath: CiscoFP.TaskStatus.Status
       description: task status
       type: String
+    description: Retrieves information about a previously submitted pending job or
+      task with the specified ID. Used for deploying.
+name: ciscofp-get-url-groups-object
+    arguments:
+    - name: id
+      required: true
```
Comment (dantavori) [inline Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.yml:2295]: seems like you broke something in the yml: "name" field is not indented correctly and you have some missing details in the "id" argument (in both cases, it was ok in my commits)
please fix and feel free to reach me on Slack if you need help.

File: Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.yml [hunk 2]
```diff
@@ -2255,10 +2279,70 @@ script:
...
+      type: string
+    - contextPath: CiscoFP.URLGroups.Objects.ID
+      description: The object ID.
+      type: string
+    - contextPath: CiscoFP.URLGroups.Objects.Type
+      description: The object type.
+      type: string
+    description: Retrieves the groups of url objects and addresses associated with
+      the specified ID. If not supplied, retrieves a list of all url objects.
+- name: ciscofp-update-url-groups-objects
```
Comment (dantavori) [inline Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.yml:2323]: same issue here

**Review by dantavori — CHANGES_REQUESTED** [review]
Please see me notes and update the form I sent you on Slack with the contribution details.

**Review by morita39 — COMMENTED** [review]
I compared original and yml part of integration downloaded from my XSOAR, but it is very difficult to check what different.
Instead, I take your word and look my code I remove the mentioned section due to duplication. also I fixed indentation.
Thank you for being patient.

regards,
Takashi

**Comment by dantavori** [comment]
I fixed again the yml file and added the commands with the examples you sent me to the readme file. Please review everything again, specifically I would like to understand if you removed the outputs of `ciscofp-update-url-groups-objects` on purpose.
When we're all set, let me know when we can schedule a quick call for a demo (let's make sure that in the demo we will use these files and not what you have locally).

**Comment by morita39** [comment]
> I fixed again the yml file and added the commands with the examples you sent me to the readme file. Please review everything again, specifically I would like to understand if you removed the outputs of `ciscofp-update-url-groups-objects` on purpose.
> When we're all set, let me know when we can schedule a quick call for a demo (let's make sure that in the demo we will use these files and not what you have locally).

I will be available to do the call. please choose . Before the call, please let me know  some instructions how to import these files in XSOAR. I tried to import the yml file , error shows up " Failed parsing file, check file format Error details yaml: line 99: mapping values are not allowed in this context"

- 22 after 8:00AM GMT
- 26 after 9:00AM GMT
- 27 after 8:00AM GMT
- 29 after 8:00AM GMT

**Comment by dantavori** [comment]
@morita39 when you tried to upload the integration to your env, were you using the most updated yml file in this branch? It should be OK.
Let's set a quick call whenever you are available to talk about it (and also about If-Then-Else). Please reach me on Slack (Dan Tavori) to schedule the call.

**Comment by dantavori** [comment]
closing - new PR https://github.com/demisto/content/pull/12425

**Pull request closed without merging**